client/wfm/src/result/StepResult.ts

```diff
@@ -0,0 +1,17 @@
+export interface StepResult {
```

Comment:
Ah, comments for the steps?
I need to try and recreate them in the demo app.

Essentially what I did so far was grabbing data from mongo, so I'm not completely sure if I'm missing some fields or have some extra that shouldn't be on the core lib.